sql/2019/04_Media/04_09b.sql

```diff
@@ -0,0 +1,17 @@
+/*
+standard SQL
+04_09b
+10 TB
+
+looks for "Accept-CH" in the body of a response.  
```

Comment:
This can be made more robust by querying the [`meta-nodes` custom metric](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js#L65).

[Here's an example of how it's queried](https://github.com/HTTPArchive/almanac.httparchive.org/blob/770dfdb34684675e1b10e2353a0d1d0eeb2705b2/sql/2019/10_SEO/10_07c.sql) using a UDF.